src/cmd/run.go

```diff
@@ -57,6 +69,41 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := "opslevel-runner-leader-lock"
+	leaseLockNamespace := viper.GetString("pod-namespace")
+	podName := viper.GetString("pod-name")
+
+	config, err := pkg.GetKubernetesConfig()
+
+	if err != nil {
+		log.Info().Msgf("Failed to get kubeconfig")
+	}
+
+	client := k8sClientset{
+		client: clientset.NewForConfigOrDie(config),
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	lock := client.getNewLock(leaseLockName, podName, leaseLockNamespace)
+	pkg.RunLeaderElection(lock, ctx, podName)
+}
+
+func (c k8sClientset) getNewLock(lockname, podname, namespace string) *resourcelock.LeaseLock {
+	return &resourcelock.LeaseLock{
+		LeaseMeta: metav1.ObjectMeta{
+			Name:      lockname,
+			Namespace: namespace,
+		},
+		Client: c.client.CoordinationV1(),
+		LockConfig: resourcelock.ResourceLockConfig{
+			Identity: podname,
```

Comment:
we should rename the var `podname` to `lockIdentity` so that its more clear what its used for.